adi/ad4630.py

```diff
@@ -166,3 +169,33 @@ def calibscale(self):
         def calibscale(self, calibscale):
             """Set calibration scale value."""
             self._set_iio_attr(self.name, "calibscale", False, calibscale, self._ctrl)
+
+
+class adaq42xx(ad4630):
+
+    """ ADAQ4224 is a 24-bit precision SAR ADC data acquisition module """
+
+    _compatible_parts = ["adaq4224", "adaq4216", "adaq4220"]
+
+    def __compat_parts__(self):
+        return adaq42xx._compatible_parts
+
+    def __init__(self, uri="", device_name="adaq4224"):
+        super().__init__()
```

Comment:
need to pass uri and device_name to super

Reply:
done